KsApi/GraphSchema.swift

```diff
@@ -57,53 +35,6 @@ public func encodeToBase64(_ input: String) -> String {
   return Data(input.utf8).base64EncodedString()
 }
 
-public struct RelayId: Decodable {
-  let id: String
-}
-
-extension RelayId: ExpressibleByStringLiteral {
-  public init(unicodeScalarLiteral value: String) {
-    self.init(id: value)
-  }
-
-  public init(extendedGraphemeClusterLiteral value: String) {
-    self.init(id: value)
-  }
-
-  public init(stringLiteral value: String) {
-    self.init(id: value)
-  }
-}
-
-public enum QueryArg<T: CustomStringConvertible> {
-  case after(String)
-  case before(String)
-  case first(Int)
-  case last(Int)
-  case arg(T)
-}
-
-public enum EdgesContainerBody<Q: QueryType> {
-  case pageInfo(NonEmptySet<PageInfo>)
-  case edges(NonEmptySet<Edges<Q>>)
-}
-
-public enum PageInfo: String {
-  case endCursor
-  case hasNextPage
-  case hasPreviousPage
-  case startCursor
-}
-
-public enum Edges<Q: QueryType> {
-  case cursor
-  case node(NonEmptySet<Q>)
-}
-
-public enum Nodes<Q: QueryType> {
-  case nodes(NonEmptySet<Q>)
-}
-
 public struct GraphResponseError: Decodable {
```

Comment:
You can delete this, the `GraphResponseErrorEnvelope` and all the methods referencing them since these were specifically created for the old approach.

Reply:
Cannot remove this yet because its' used by `GraphError` which is used by `ErrorEnvelope` which is used by every endpoint in `Service` (v1 and apollo).